jboss/container/wildfly/launch/datasources/test/datasource/expectations/datasources_ibmdb2_xa.xml

```diff
@@ -15,12 +15,6 @@
         <driver name="h2" module="com.h2database.h2">
             <xa-datasource-class>org.h2.jdbcx.JdbcDataSource</xa-datasource-class>
         </driver>
-        <driver name="mysql" module="com.mysql">
```

Comment:
is the changes in the config related to GitHub actions?

Reply:
No, these tests needed to be updated to reflect that the config we're using to test doesn't have the mysql or postgres drivers in them anymore.